cmd/kubeadm/app/apis/kubeadm/v1beta2/doc.go

```diff
@@ -19,6 +19,7 @@ limitations under the License.
 // +k8s:deepcopy-gen=package
 // +k8s:conversion-gen=k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm
 
+// Package v1beta2 has been deprecated by v1beta3
```

Comment:
please add an empty `//` line after this one to make it more visible in these generated docs https://kubernetes.io/docs/reference/config-api/kubeadm-config.v1beta2/